tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -710,6 +721,36 @@ def validate_header_rpt_month_year(datafile, header, generate_error):
         )
     return is_valid, error
 
+
+def t3_child_validator(which_child):
```

Comment:
TODO: should change this validator name as well as returning error messages, since this can be used in M3 and Tribal T3